src/main/java/genesis/Genesis.java

```diff
@@ -0,0 +1,530 @@
+package genesis;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a generic task.
+ */
+
+
+class Task {
```

Comment:
Would it be better to separate the different classes into different files instead?